src/index.js

```diff
@@ -142,17 +142,30 @@ export function app(state, actions, view, container) {
     return node ? node.key : null
   }
 
+  function eventListener(event) {
+    return event.target.events[event.type](event)
+  }
+
   function updateAttribute(element, name, value, oldValue, isSVG) {
     if (name === "key") {
     } else if (name === "style") {
       for (var i in clone(oldValue, value)) {
         element[name][i] = value == null || value[i] == null ? "" : value[i]
       }
     } else {
-      if (
-        typeof value === "function" ||
-        (name in element && name !== "list" && !isSVG)
-      ) {
+      if (name[0] === "o" && name[1] === "n") {
```

Comment:
This will add lifecycle events and any other properties starting with "on" to the `element.events` object, right?   Any downside to this?  I suspect not, but thought it was worth asking.

Reply:
Yes, you are right. I can't think of any downsides, can you?

* oncreate/onupdate/onremove/ondestroy: Will get added too, although never used. However, this is still better than our _current_ behavior, where we add each of them them directly on the element.

Reply:
I agree, it's probably a non-issue.